fission-web-server/library/Fission/Web/Server/Internal/Tasks.hs

```diff
@@ -45,7 +54,71 @@ deleteByUsername userNameTxt =
         Nothing                -> error "User doesn't exist"
 
 ---
+syncDNS :: Server()
```

Comment:
💅 Purely aesthetic:

In Haskell, `()` is a type on its own. We put a space between data types

```haskell
-- Looks super weird
Server()

-- Fails to parse
MaybeInt

-- Looks good
Server ()
Maybe Int
```

Reply:
Perhaps to clarify the role of `()` 👇 

This is called "unit", and is has the type `() :: ()` (type-level punning makes the term and type read the same in this case). It has only the one value: `()`, in the same way that `Bool` has two values.

```haskell
data () = ()
data Bool = False | True
```

`()` carries very little information. As such, it can be used to stub out a type. For example, these types carry the same data (though they are counted as different by the compiler):

```haskell
Maybe a ~ Either () a
```

Since it carries no information of its own, we can always get a unit. For example, this would be totally valid (though stupid, but maybe good for building intuition):

```haskell
ignore :: a -> ()
ignore _ = ()

ignore 42
-- ()

ignore "this is a string"
-- ()

ignore ()
-- ()

ignore (Just (Just (Right "hi")))
-- ()
```

Being FP, we always need to return a value, so especially when running side effects with no return value of their own, we use `()` to represent that "nothing" was returned.